app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,34 @@
 class SessionsController < ApplicationController
   def create
+    user_info = request.env['omniauth.auth']
+    name = user_info[:info][:name]
+    uid = user_info[:uid]
+    email = user_info[:info][:email]
+    token = user_info[:credentials][:token]
+    
+    #found_user = User.find_or_create_by(uid: uid, email: email, token: token, name: name)
+
+    found_user = User.find_or_create_by(name: name) do |user|
+      user.uid = uid
+      user.email = email
+      user.token = token
```

Comment:
Is this token static or does it change every time you log in? If it's the latter wouldn't we want to store it in the session?

Reply:
Throughout today my token has been static. I'll monitor it tomorrow as well and see if it changes!